src/main/java/uk/ac/ebi/ena/webin/cli/service/IgnoreErrorsService.java

```diff
@@ -58,15 +66,24 @@ public IgnoreErrorsRequest(String context, String name) {
     private boolean getIgnoreErrors(String userName, String password, String context, String name, boolean test) {
 
         RestTemplate restTemplate = new RestTemplate();
-        restTemplate.setErrorHandler(new DefaultErrorHander(WebinCliMessage.IGNORE_ERRORS_SERVICE_SYSTEM_ERROR.text()));
```

Comment:
I think we should continue to use setErrorHandler and improve the NotFoundErrorHandler. See the previous comment.